src/proof/alf/alf_printer.cpp

```diff
@@ -629,6 +629,10 @@ std::string AlfPrinter::getRuleName(const ProofNode* pfn) const
   {
     ProofRewriteRule id;
     rewriter::getRewriteRule(pfn->getArguments()[0], id);
+    if (id == ProofRewriteRule::STR_IN_RE_EVAL && options().proof.proofCpcStrInReNfa)
```

Comment:
plz make sure your changes in the h and cpp files follow the formatting guidelines:
https://github.com/cvc5/cvc5/wiki/Developer-Guide#formatting-with-clang-format